db/schema.rb

```diff
@@ -430,7 +430,6 @@
     t.datetime "opening_at"
     t.datetime "closing_at"
     t.integer  "corporate_information_page_type_id",          limit: 4
-    t.string   "need_ids",                                    limit: 255
```

Comment:
So I deleted this entry, as I thought this might convince ActiveRecord not to select it from the database. This is probably wrong, so I've removed the commit deleting this line.